cityofaustin/atd-moped
# Show signal ID in component list item

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/11496

This PR adds the signal ID the component list item text. I also reworked the text styles, so that component name and subtype are always the primary text. We're going to keep refining these styles as we add more component attributes, e.g. [component phasing](https://github.com/cityofaustin/atd-data-tech/issues/11429), so we'll revisit this again soon.

## Testing
**URL to test:** [Netlify](https://deploy-preview-982--atd-moped-main.netlify.app/)

**Steps to test:**
1. Create a traffic and phb signal component.
2. Observe the signal ID and location name in the secondary list item text (muted gray)
3. Create other components.
4. Observe that the component name and subtype are now shown in the primary list item text.

---
#### Ship list
